app/components/Settings/OptionsDropdown/OptionsDropdown.react.js

```diff
@@ -1,61 +1,46 @@
 import React, {Component} from 'react';
 import PropTypes from 'prop-types';
-import {flatten, keys} from 'ramda';
-import Select from 'react-select';
+import SQLOptions from './SQLOptions.react';
+import ESIndicesOptions from './ESIndicesOptions.react';
+import ESDocsOptions from './ESDocsOptions.react';
 
 export default class OptionsDropdown extends Component {
-    constructor(props) {
-        super(props);
-        this.renderSQLOptions = this.renderSQLOptions.bind(this);
-        this.renderElasticsearchIndecies = this.renderElasticsearchIndecies.bind(this);
-        this.renderElasticsearchDocs = this.renderElasticsearchDocs.bind(this);
-    }
 
     static propTypes = {
         // See type of react-select's Select `value` property
         selectedTable: PropTypes.any,
         selectedIndex: PropTypes.any,
-
```

Comment:
I think this line was here to make clear the above comment only applies to `selectedTable` and `selectedIndex`.